pages/services/cassandra/2.6.0-3.11.4/getting-started/index.md

```diff
@@ -0,0 +1,187 @@
+
+   The slashes in your service name are interpreted as folders. You are deploying {{ model.techShortName }} in the `/testing` folder. Any user with access to the `/testing` folder will have access to the service.
+
+<p class="message--note"><strong>NOTE: </strong>Services cannot be renamed. Because the location of the service is specified in the name, you cannot move services between folders. DC/OS 1.9 and earlier does not accept slashes in service names. You may be able to create the service, but you will encounter unexpected problems.</p>
+
+### Interacting with your foldered service
+
+- Interact with your foldered service via the DC/OS CLI with this flag: `--name=/path/to/myservice`.
+- To interact with your foldered service over the web directly, use `http://<dcos-url>/service/path/to/myservice`. E.g., `http://<dcos-url>/service/testing/{{ model.serviceName }}/v1/endpoints`.
+
+# Multi-datacenter Deployment
+
+To replicate data across data centers, Apache {{ model.techShortName }} requires that you configure each cluster with the addresses of the seed nodes from every remote cluster. Here's what starting a multi-data-center Apache {{ model.techShortName }} deployment would look like, running inside of a single DC/OS cluster.
+
+## Launch two {{ model.techShortName }} clusters
+
+Launch the first cluster with the default configuration:
+
+```shell
+dcos package install {{ model.serviceName }}
+```
+
+Create an `options.json` file for the second cluster that specifies a different service name and data center name:
+
+```json
+{
+  "service": {
+    "name": "{{ model.serviceName }}2",
+    "data_center": "dc2"
+  }
+}
+```
+
+Launch the second cluster with these custom options:
+```
+dcos package install {{ model.serviceName }} --options=<options>.json
+```
+
+## Get the seed node IP addresses
+
+<p class="message--note"><strong>NOTE: </strong>If your {{ model.techShortName }} clusters are not on the same network, you must set up a proxying layer to route traffic.</p>
+
+Get the list of seed node addresses for the first cluster:
+
+```shell
+dcos {{ model.serviceName }} endpoints node
+```
+
+Alternatively, you can get this information from the scheduler HTTP API:
+
+```json
+DCOS_AUTH_TOKEN=$(dcos config show core.dcos_acs_token)
+DCOS_URL=$(dcos config show core.dcos_url)
+curl -H "authorization:token=$DCOS_AUTH_TOKEN" $DCOS_URL/service/{{ model.serviceName }}/v1/endpoints/node
+```
+
+Your output will resemble:
+
+```
+{
+  "address": [
+    "10.0.1.236:9042",
+    "10.0.0.119:9042"
+  ],
+  "dns": [
+    "node-0-server.{{ model.serviceName }}.autoip.dcos.thisdcos.directory:9042",
+    "node-1-server.{{ model.serviceName }}.autoip.dcos.thisdcos.directory:9042"
+  ],
+  "vip": "node.{{ model.serviceName }}.l4lb.thisdcos.directory:9042"
+}
+```
+
+Note the IP addresses in the `address` field.
+
+Run the same command for your second {{ model.techShortName }} cluster and note the IP addresses in the `address` field:
```

Comment:
@viivek46 Can you mention the command here instead of using `same command` for more clarity,

Reply:
@anithamuthu79  same command is mentioned because it is same as above and commands are already mentioned below.